zhaquirks/xiaomi/__init__.py

```diff
@@ -563,9 +564,17 @@ def __init__(self, *args, **kwargs):
         self.endpoint.device.consumption_bus.add_listener(self)
         self.endpoint.device.power_bus.add_listener(self)
 
+        # put a default value so the sensors are created
+        if self.POWER_ID not in self._attr_cache:
+            self._update_attribute(self.POWER_ID, 0)
+        if self.VOLTAGE_ID not in self._attr_cache:
+            self._update_attribute(self.VOLTAGE_ID, 0)
+        if self.CONSUMPTION_ID not in self._attr_cache:
+            self._update_attribute(self.CONSUMPTION_ID, 0)
+
     def power_reported(self, value):
         """Power reported."""
-        self._update_attribute(self.POWER_ID, value)
+        self._update_attribute(self.POWER_ID, value * 10)
```

Comment:
This change made my another aqara outlet (mmeu01 square one) report 10* more usage than normal

Reply:
That should be fixed by either re-pairing the device or reading the "ac_power_divisor" attribute on the ElectricalMeasurement cluster through ZHA -> Devices -> Plug -> Manage Clusters -> ElectricalMeasurement -> ac_power_divisor -> read attribute.

Reply:
Reparing is workaround, I don't think this is good user experience to ask user to re-pair working device,
I think the solution is to remove this multiplication and change ac_power_divisor back to 1, I've changed it locally and both devices shows power consumption correctly for me

Reply:
I agree. If possible, the attribute should be re-read automatically. There is another issue with it being left on 1 (different rounding when values like 1.9W are reported which causes a jump from 2W to 1W after some time with unchanged consumption).

If consumption varies a lot when low, it's also nice to see values in between 0W and 1W (and it not constantly jumping around).

Reply:
The power measurement on LLKZMK11LM (lumi.relay.c2acn01) is also wrong after this change. It's showing 100W for a 10W device.

Reply:
@hakong No, check the AC power divisor: ``0x0605``. It's set to ``10`` instead of ``1`` now. Both of these changes were made so that there's one decimal.